data/sea/05-error.h

```diff
@@ -121,6 +123,19 @@ static ierror_msg_t NOINLINE ierror_loc_pretty (ierror_loc_t loc, iint_t line)
     char *p  = msg_text;
     char *pe = msg_text + msg_size;
 
+    char *filename = calloc (255, 1);
+    char *proc     = calloc (269, 1);
```

Comment:
is the intention here not to bother freeing the memory because it's only a temporary fix?